controllers/tokenupload_controller.go

```diff
@@ -53,9 +53,9 @@ var uploadSecretSelector = metav1.LabelSelector{
 	},
 }
 
-// +kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;delete
-// +kubebuilder:rbac:groups="",resources=secrets,verbs=get;watch;create;update;list;delete
-// +kubebuilder:rbac:groups="",resources=serviceaccounts,verbs=get;list;watch;create;update;delete
+//+kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;delete
+//+kubebuilder:rbac:groups="",resources=secrets,verbs=get;watch;create;update;list;delete
+//+kubebuilder:rbac:groups=appstudio.redhat.com,resources=remotesecrets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Token upload controller just declared incorrect permissions that it requires. This makes no difference because kubebuilder sums all the reqs of all the controllers together, but I think we should just be precise about what each controller requires.